tasks/deploy/extension.ts

```diff
@@ -1,11 +1,21 @@
+import { LedgerSigner } from "@anders-t/ethers-ledger";
 import { task } from "hardhat/config";
 import { TaskArguments } from "hardhat/types";
 
-import { deploy } from "../helpers";
+import { deploy, deployWithLedger } from "../helpers";
 
 task("deploy:extension")
   .addParam("name", "Extension contract name to deploy. Must be constructorless.")
   .setAction(async function (taskArguments: TaskArguments, { ethers }) {
     const contract = await deploy(ethers, taskArguments.name);
     console.log(`${taskArguments.name} deployed to: `, contract.address);
   });
+
+task("hd:deploy:extension", "Deploys all extensions needed for Humanbound token")
+  .addParam("name", "Extension contract name to deploy. Must be constructorless.")
```

Comment:
```suggestion
  .addParam("extensionName", "Extension contract name to deploy. Must be constructorless.")
```

Reply:
So turns out hardhat doesn't like parameter names that have uppercase characters and I'm not a fan of `extensionname`, so I'd prefer `name` out of brevity and simplicity as the context is already fairly clear what this name is of, especially with the parameter description.

Thoughts?